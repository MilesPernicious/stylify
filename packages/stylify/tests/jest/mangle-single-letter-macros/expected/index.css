._t1f4c{
	d:'M 850'
}
._t1g2d{
	d:'M 851'
}
._rsml{
	margin:24px
}
._rtkm{
	margin:25px
}

@media (min-width: 768px) {
._3hirn{
	d:'M 850'
}
._3hjpo{
	d:'M 851'
}
._t7rr0p:hover{
	d:'M 850'
}
._t7rq2o:hover{
	d:'M 851'
}
._0bcg0{
	margin:24px
}
._0bbhv{
	margin:25px
}
._s2lcg:hover{
	margin:24px
}
._s2kef:hover{
	margin:25px
}
}