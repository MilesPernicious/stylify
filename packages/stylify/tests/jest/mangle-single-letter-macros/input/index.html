<div class="d:^M__850^"></div>
<div class="m:24px"></div>
<div class="d:^M__851^ m:25px"></div>
<div class="m:25px d:^M__851^"></div>

<div class="md:d:^M__850^"></div>
<div class="md:m:24px"></div>
<div class="md:d:^M__851^ md:m:25px"></div>
<div class="md:m:25px md:d:^M__851^"></div>

<div class="md:hover:d:^M__850^"></div>
<div class="md:hover:m:24px"></div>
<div class="md:hover:d:^M__851^ md:hover:m:25px"></div>
<div class="md:hover:m:25px md:hover:d:^M__851^"></div>
