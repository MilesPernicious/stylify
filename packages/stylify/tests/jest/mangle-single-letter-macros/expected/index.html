<div class="_t1f4c"></div>
<div class="_rsml"></div>
<div class="_t1g2d _rtkm"></div>
<div class="_rtkm _t1g2d"></div>

<div class="_3hirn"></div>
<div class="_0bcg0"></div>
<div class="_3hjpo _0bbhv"></div>
<div class="_0bbhv _3hjpo"></div>

<div class="_t7rr0p"></div>
<div class="_s2lcg"></div>
<div class="_t7rq2o _s2kef"></div>
<div class="_s2kef _t7rq2o"></div>
